src/client/App.css

```diff
@@ -0,0 +1,2 @@
+/* If your css doesn't override the default try using the :important rule */
```

Comment:
`!important` should generally be avoided. Is this instruction provided to give the user a way to override your inline styles?